docpages/example_programs/the_basics/editing-channel-and-message.md

```diff
@@ -0,0 +1,89 @@
+\page editing-channel-and-message Editing Channel and Message
```

Comment:
I would change it to `Editing Channels and Messages` and `editing-channels-and-messages`. Remember to change it in `the_basics.md`! This comes from a discussion from a previous comment.

Reply:
Plural as suggested by braindigitals